curriculum/challenges/english/01-responsive-web-design/basic-css/add-borders-around-your-elements.english.md

```diff
@@ -107,8 +107,27 @@ tests:
 
 ## Solution
 <section id='solution'>
+  
+To add borders to the element, you must first create the class somewhere within the `<style>` tags:
 
-```js
-// solution required
+```html
+.thick-green-border {
+  border-width: 10px;
+  border-color: green;
+  border-style: solid;
+}
+```
+Or:
+
+```html
```

Comment:
github does support css highlighting, so it'd look better if you used that instead of html.

https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L606